tv/pages/index.tsx

```diff
@@ -62,14 +62,17 @@ const defaultSchedule: ScheduleSlot[][] = [];
 const Schedule = (props: ScheduleProps): JSX.Element => {
   const displayedDaySessions = useRef(props.sessions);
   const [schedule, setSchedule] = useState(defaultSchedule);
-  const dayTimeContext = useContext(DayTimeContext);
+  const { dayTime } = useContext(DayTimeContext);
+  const dayTimeContext = useRef(useContext(DayTimeContext));
```

Comment:
I am not a React expert and I do not understand why we need a `useRef` around a new `useContext` of the same `DayTimeContext` context as the previous line. Can you explain to me why is it needed and why we cannot simply use the previous `const dayTimeContext = useContext(DayTimeContext);`?